group-coordinator/src/test/java/org/apache/kafka/coordinator/group/streams/MockTaskAssignor.java

```diff
@@ -60,6 +65,7 @@ public String name() {
     @Override
     public GroupAssignment assign(final GroupSpec groupSpec, final TopologyDescriber topologyDescriber)
         throws TaskAssignorException {
+        assignmentConfigs = groupSpec.assignmentConfigs();
```

Comment:
There is no changes to `GroupSpec` in this PR, so not sure if understand the relationship to the rest of the code

Reply:
We are using `lastPassedAssignmentConfigs` to test which configs are passed to the assignor. The `assignmentConfigs` were already there in the `GroupSpec`, not changes were required. The Mockassignor was just ignoring the map.